webhooks/ssp_webhook.go

```diff
@@ -157,8 +157,9 @@ func (s *sspValidator) validateOperandPlacement(ctx context.Context, namespace s
 				Spec: v1.PodSpec{
 					Containers: []v1.Container{
 						{
-							Name:  podName,
-							Image: naImage,
+							Name:                     podName,
+							Image:                    naImage,
+							TerminationMessagePolicy: v1.TerminationMessageFallbackToLogsOnError,
```

Comment:
This change is not needed, because this call does not create the deployment, it is a dry-run just to check if the placement fields are valid.